packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -46,6 +46,10 @@ export interface TooltipProps {
    * It sets a max-width for the Tooltip
    */
   maxWidth?: number | CSS.Property.MaxWidth;
+  /**
+   * It sets a delay period for the Tooltip
+   */
+  delay?: Partial<DelayProps>;
```

Comment:
IMO, having one prop for each delay is a better API than passing an object to a prop. For two reasons:

1. Comparing object is a bit weird
2. Easier to use if I only want to set one of the delays
`<Tooltip showDelay={100} />` 
`<Tooltip hideDelay={100} />` 
`<Tooltip showDelay={100} hideDelay={100} />`